### ISSUE TITLE
[Bug]: Fail Fast when add auth

### ISSUE THREAD

**Issue by gyliu513** [k8sgpt-ai/k8sgpt#1170]
### Checklist

- [X] I've searched for similar issues and couldn't find anything matching
- [X] I've included steps to reproduce the behavior

### Affected Components

- [X] K8sGPT (CLI)
- [ ] K8sGPT Operator

### K8sGPT Version

v0.3.37

### Kubernetes Version

v1.21

### Steps to reproduce

1. add openai auth with an invalid api key
2. run k8sgpt with `--explain` to trigger AI backend
3. got error due to openai api key is not right

### Expected behaviour

Can k8sgpt report errors when adding an invalid auth?

### Actual behaviour

No error with invalid auth, and customer will got errors when running k8sgpt explain.

**Comment by matthisholleville** [comment]
Hello! Do you have any ideas or suggestions for implementation? I think it would be useful to have an extra parameter (e.g., `k8sgpt auth add .... --check`), but it seems that not all providers (including OpenAI) offer a route to test if an API key is valid without opening a client, etc.

**Comment by gyliu513** [comment]
@matthisholleville yes, I have same thinking as you, how about just call a simple `generate` API with prompt as `hello world` for different vendor platforms?

**Comment by AlexsJones** [comment]
Closing as stale. No activity in 6+ months and no repro on current k8sgpt v0.4.36 / operator v0.2.27. Please reopen with a current-version repro if this is still happening.
